### DISCUSSION TITLE
Device got 10.42.42.42 but failed to backup

### DISCUSSION THREAD

**Discussion by JAAlperin** [ct-Open-Source/tuya-convert#1051] [Q&A]
I accidentally said no to "continue anyway".  Device (Gosund WP5) cannot be reset or put into pairing mode.  Any way to complete the flashing or salvage the device?

**Comment by kueblc** [comment]
Power cycle the device and ignore the prompt to put the device in pairing mode. The flashing script should pick up where it left off.
